### DISCUSSION TITLE
Is Frog.signature not working?

### DISCUSSION THREAD

**Discussion by salviega** [wevm/frog-archived#459] [General]
I am trying to sign a message with Metamask but the docs's example generate error. Look at:

![code](https://github.com/user-attachments/assets/2c4bf974-b6ee-4375-b79b-65058d962625)

![Screenshot 2024-08-03 at 12 24 21](https://github.com/user-attachments/assets/70378a31-a618-4aee-a705-06795e4f3d38)

![Screenshot 2024-08-03 at 12 24 51](https://github.com/user-attachments/assets/2d7ba62b-d2aa-4f17-a997-b6c62085a9bd)

**Comment by dalechyn** [comment]
Hey! Converting to an issue, I think we aren't supporting Signature Handler in the Frog debugger yet.

**Reply by dalechyn** [reply]
Yeah it needs to be implemented, #429
